orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -564,13 +599,57 @@ def _enable_conditional_optimizations(
                 # Enable sparsity-based optimization when applicable.
                 if len(label_sparsity_results) > 0:
                     graph_transformer_config.sparse_label_input_names = label_sparsity_results
-                    logger.info(f"Label sparsity based optimization is on for {label_sparsity_results}")
+                    self._logger.info("Label sparsity based optimization is on for %s", label_sparsity_results)
```

Comment:
nit: ON
Also, can we added the sparsity percentage? Example: 95%. Not sure if that data is easily accessible here.

Reply:
Have printed the density of input if we plan to enable opt for the input. :)